lib/js/src/manager/file/_FileManagerBase.js

```diff
@@ -230,6 +230,25 @@ class _FileManagerBase extends _SubManagerBase {
             return isInRemoteFiles && isInEphemeralFiles;
         }
     }
+
+
+    /**
+     * Check if an SdlFile needs to be uploaded to Core or not.
+     * It is different from hasUploadedFile() because it takes isStaticIcon and overwrite properties into consideration.
+     * ie, if the file is static icon, the method always returns false.
+     * If the file is dynamic, it returns true in one of these situations:
+     * 1) the file has the overwrite property set to true
+     * 2) the file hasn't been uploaded to Core before.
+     *
+     * @param {SdlFile} sdlFile - SdlFile with file name and one of A) fileData, B) Uri, or C) resourceID set
```

Comment:
```suggestion
     * @param {SdlFile} file - SdlFile with file name and one of A) fileData, B) Uri, or C) resourceID set
```
Fixes issues found by linter.